packages/forma-36-tokens/tools/build.js

```diff
@@ -71,16 +71,15 @@ const buildScssTokens = (srcPath, tokens) => {
 };
 
 const buildIndexJS = (srcPath, tokens) => {
-  return fse.outputFile(
-    srcPath,
-    `
-    Object.defineProperty(exports, "__esModule", {
-      value: true
-    });
+  const objectLiteral = JSON.stringify(tokens, null, 2);
+  const cjs = `Object.defineProperty(exports, "__esModule", { value: true });\nconst tokens = ${objectLiteral};\nmodule.exports = tokens;\nmodule.exports.default = tokens;\n`;
+  return fse.outputFile(srcPath, cjs);
+};
 
-    module.exports = ${JSON.stringify(tokens, null, 2)}
-    `,
-  );
+const buildIndexMJS = (srcPath, tokens) => {
+  const objectLiteral = JSON.stringify(tokens, null, 2);
+  const esm = `// ESM build for @contentful/f36-tokens\nconst tokens = ${objectLiteral};\nexport default tokens;\nexport { tokens };\n`;
+  return fse.outputFile(srcPath.replace(/index\.js$/, 'index.mjs'), esm);
```

Comment:
Let’s replace this line with `return fse.outputFile(srcPath, mjs);`

Reply:
No, the current version is a hack that gets `'something/index.js'` as input and then replaces the extension with `.mjs`. Passing `'something/index.mjs'` directly has the same outcome and we can remove the hack.